src/lib/rule-finder.js

```diff
@@ -108,6 +108,6 @@ function RuleFinder(specifiedFile, options) {
   this.getUnusedRules = () => getSortedRules(unusedRules);
 }
 
-module.exports = function (specifiedFile, options = {}) {
+module.exports = function (specifiedFile, options) {
```

Comment:
it should be specified in the exported place then; since that's what consumers see.